conf/pipeline/sarek/crukmi.config

```diff
@@ -0,0 +1,33 @@
+// Profile config names for nf-core/configs
+
+params {
+  // Specific nf-core/configs params
+  config_profile_description = 'Cancer Research UK Manchester Institute HPC cluster profile provided by nf-core/configs'
+  config_profile_contact = 'Stephen Kitcatt, Simon Pearce (@skitcattCRUKMI, @sppearce)'
+  config_profile_url = 'http://scicom.picr.man.ac.uk/projects/user-support/wiki'
+}
+
+// Specific nf-core/sarek process configuration
+process {
+
+  withName: 'SAMTOOLS_MPILEUP' {
+    cpus   = 1
+    memory = { check_resource( 5.GB * task.attempt) }
+  }
+
+}
+
+def check_resource(obj) {
+    try {
+      if (obj.getClass() == nextflow.util.MemoryUnit && obj.compareTo(params.max_memory as nextflow.util.MemoryUnit) == 1)
+        return params.max_memory as nextflow.util.MemoryUnit
+      else if (obj.getClass() == nextflow.util.Duration && obj.compareTo(params.max_time as nextflow.util.Duration) == 1)
+        return params.max_time as nextflow.util.Duration
+      else if (obj.getClass() == java.lang.Integer)
+        return Math.min(obj, params.max_cpus as int)
+      else
+        return obj
+    } catch (all) {
+        println "   ### ERROR ###   Max params max_memory:'${params.max_memory}', max_time:'${params.max_time}' or max_cpus:'${params.max_cpus}'  is not valid! Using default value: $obj"
+    }
+}
```

Comment:
You should not need that part with the latest sarek release

Reply:
I just copied it from `cfc.config` ;)

Reply:
I'll have a look at what they're doing to help them update that as well.